docs/how-to-help-with-video-challenges.md

```diff
@@ -122,7 +122,7 @@ More answers
 1
 ```
 
-Update the  “Question Text” with your question. Update the `Answer 1`, `Answer 2`, and so on with the possible answers. Make sure to update the video-solution number with the correct answer number. You can add more possible answers using the same format. The question and answers can be surrounded by quotation marks.
+Update the  “Question Text” with your question. Update the `Answer 1`, `Answer 2`, and so on with the possible answers. Make sure to update the `video-solution` number with the correct answer number. You can add more possible answers using the same format. The question and answers can be surrounded with quotation marks.
```

Comment:
@nhcarrigan What do you think about the following or something like it?  I was trying to avoid "Question Text".
```suggestion
Add/Update the  question under `# --question--` `## --text--`. Add/Update answers (`Answer 1`, `Answer 2`, and so on) under `## --answers--`. Make sure to update the number under `## --video-solution--` with the correct answer number. You can add more possible answers using the same format. The question and answers can be surrounded with quotation marks.
```